src/org/jitsi/impl/neomedia/rtp/translator/RTPTranslatorImpl.java

```diff
@@ -719,9 +719,9 @@ public GlobalTransmissionStats getGlobalTransmissionStats(
      */
     public long getLocalSSRC(StreamRTPManager streamRTPManager)
     {
-        // if (streamRTPManager == null)
-        //    return localSSRC;
-        // return ((RTPSessionMgr) manager).getLocalSSRC();
+         if (streamRTPManager == null)
```

Comment:
I'd add a longer comment up here which explains why the assumption below was causing issues and what this is addressing (getting different SSRCs at different points in time).  I don't think I follow the comment you added below.

Reply:
Also I think we need to adjust the comment on `localSSRC` at the top, as it documents `localSSRC` as overriding the SSRC rom the RTP manager, but now it's the opposite.